portal-impl/src/com/liferay/portal/upgrade/v7_0_3/UpgradeOrganization.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.portal.upgrade.v7_0_3;
+
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+import com.liferay.portal.kernel.util.LoggingTimer;
+
+/**
+ * @author Manuel de la Peña
+ */
+public class UpgradeOrganization extends UpgradeProcess {
+
+	@Override
+	protected void doUpgrade() throws Exception {
+		updateRegularOrganizations();
+	}
+
+	protected void updateRegularOrganizations() throws Exception {
+		try (LoggingTimer loggingTimer = new LoggingTimer()) {
+			runSQL("update Organization_ set type_=\"organization\"");
```

Comment:
Hey @mdelapenya @pei-jung,

So what happens with this line to all of our customers who are using organization types for their own purposes?

For example, an education portal may be using organization types such as "School Group", "School", "Degree", "Class", ... and suddenly they all become "organization". I can imagine them being very unhappy about such a change being done massively to their database.